src/react/MODUKBody/MODUKBody.tsx

```diff
@@ -0,0 +1,13 @@
+'use client'
+
+import clsx from 'clsx'
+import { type JSX, useEffect, useState } from 'react'
+
+export const MODUKBody = ({ children, className }: JSX.IntrinsicElements['body']) => {
+  const [isClient, setClient] = useState(false)
+  useEffect(() => {
+    setClient(true)
+  }, [])
+  // logical OR is in place to prevent empty class attribute
+  return <body className={clsx(isClient && 'js-enabled', className) || undefined}>{children}</body>
+}
```

Comment:
I'm a bit concerns with the MutationObserver and the unpredictable race conditions we may have  in the new version of govuk-frontend doing the JS test using `body.className`